packages/msw-addon/package.json

```diff
@@ -8,7 +8,7 @@
   ],
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/itaditya/msw-storybook-addon.git",
+    "url": "git+https://github.com/mswjs/msw-storybook-addon.git",
```

Comment:
@kettanaito should we also update package version to v2 considering that there's a breaking change?

Reply:
I'd rather merge this and rewrite the source to TypeScript and then release it altogether. There may be more breaking changes introduced by this process. I know it's not very incremental, but the library's scope is tiny and we may get away with accumulating the changes without releasing them.